<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>info</class>
 <widget class="QMainWindow" name="info">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>634</width>
    <height>480</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>PointingHandCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Ristorante Emilia</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>641</width>
      <height>431</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(/Users/emanuelefaraglia/Downloads/Sfondo.jpg);
background-position: center;
background-repeat: no-repeat;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>267</x>
      <y>380</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
        border: 2px solid #5A5A5A;
        border-radius: 10px;
        padding: 5px;
        background-color: #F0F0F0;
        color: #5A5A5A;
    }
    QPushButton:hover {
        background-color: #D3D3D3;
    }
    QPushButton:pressed {
        background-color: #A9A9A9;
    }</string>
    </property>
    <property name="text">
     <string>Indietro</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>212</x>
      <y>20</y>
      <width>210</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Apple Chancery</family>
      <pointsize>36</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Informazioni</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>90</y>
      <width>514</width>
      <height>250</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Apple Chancery</family>
      <pointsize>18</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextEdit {
    background-color: #D3CFC4;
    color: black;
    border: 1px solid black;  /* Bordo del pulsante */
}

QTextEdit:hover {
    background-color: lightgray;  /* Cambia colore al passaggio del mouse */
}</string>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Apple Chancery'; font-size:18pt; font-weight:400; font-style:italic;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Benvenuti al Ristorante Emilia, dove la tradizione culinaria incontra il fascino del mare. Situato direttamente sulla spiaggia, offriamo ai nostri ospiti un'esperienza unica tra i sapori mediterranei . I nostri piatti sono preparati con ingredienti freschissimi, portando in tavola il meglio della cucina di mare. Che si tratti di un pranzo rilassante o di una cena romantica al tramonto, il nostro ambiente accogliente e il servizio attento faranno di ogni visita un momento indimenticabile.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="salva">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>380</y>
      <width>0</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
        border: 2px solid #5A5A5A;
        border-radius: 10px;
        padding: 5px;
        background-color: #F0F0F0;
        color: #5A5A5A;
    }
    QPushButton:hover {
        background-color: #D3D3D3;
    }
    QPushButton:pressed {
        background-color: #A9A9A9;
    }</string>
    </property>
    <property name="text">
     <string>Salva</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>634</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
